### PR TITLE
Add namespace to metrics

### PR DESCRIPTION
**Pull request by ktilcu**
This will allow callers to add a global namspace for each instance of
the metrics object.

### CODE REVIEW

File: metrics.js [hunk 1]
```diff
@@ -28,23 +28,26 @@
       return true
     }
 
+    function scope (name, value) {
```
Comment (dkchou) [inline metrics.js]: Is the value parameter supposed to be here?

Comment (ktilcu) [inline metrics.js]: nope

**Comment by dkchou** [comment]
Other comments from our conversation:
1) ; line endings (Better to have those at the end of lines)?
2) 'strict mode' => 'use strict'

**Comment by dkchou** [comment]
Looks good.  :+1:

**Pull request merged**